cocos/2d/CCFontFreetype.cpp

```diff
@@ -44,7 +44,25 @@ THE SOFTWARE.
 #define SCALE_BY_DPI(x) (int)(x)
 #endif
 
-#define FFT_SDF_TMP_VECTOR 1
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_IOS) && defined(__IPHONE_OS_VERSION_MIN_REQUIRED)
```

Comment:
Why add this line `defined(__IPHONE_OS_VERSION_MIN_REQUIRED)`?

Reply:
It is redundant and removed